services/postgres/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.6-alpine
```

Comment:
I think the folder shall NOT be called ``postgres`` . It gives the impression that this is to build the postgres service and it is not!
Please rename ``service/postgres`` to e.g. ``postgres-migrator`` or ``pg-migrator``  instead

Reply:
you are right and it is not a service but the postgres-database package packaged in a docker container..
so I moved it there.